rescript/ncbi.py

```diff
@@ -88,87 +114,216 @@ def get_ncbi_data(
     return seqs, taxa
 
 
-def _get(params, ids=None, entrez_delay=0.):
-    if ids:
-        assert len(ids) >= 1, "need at least one id"
-        epost = 'https://eutils.ncbi.nlm.nih.gov/entrez/eutils/epost.fcgi'
-        data = {'db': params['db'], 'id': ','.join(ids)}
+def _get_logger(logging_level):
+    logger = logging.getLogger('rescript')
```

Comment:
Thanks @misialq, this is an ugly [workaround](https://github.com/joblib/joblib/issues/634#issuecomment-367321025) because [joblib hijacks the root logger](https://github.com/joblib/joblib/issues/692). I wish it were otherwise, but this is the only way I could find that works. I'm still missing logging from the requests library as a result. If they ever fix that issue I will be happy to refactor.

Reply:
Oh, I see. That's a shame. Let's leave it as is then 👍